TidepoolKit/Internal/Responses/DataResponse.swift

```diff
@@ -8,9 +8,9 @@
 
 struct DataResponse: Codable {
     var data: [TDatum]
-    var malformed: [Int: [String: Any]]
+    var malformed: [String: [String: Any]]
 
-    init(data: [TDatum] = [], malformed: [Int: [String: Any]] = [:]) {
+    init(data: [TDatum] = [], malformed: [String: [String: Any]] = [:]) {
```

Comment:
[nit] Is this `[String: [String: Any]]` structure a common "malformed" type?  Perhaps consider a typealias for it?